pkgs/test/lib/src/runner/browser/browser.dart

```diff
@@ -134,7 +135,7 @@ abstract class Browser {
     // For example this happens with Chrome Headless.
     // See SDK issue: https://github.com/dart-lang/sdk/issues/31264
     for (var stream in _ioSubscriptions) {
-      stream.cancel();
+      unawaited( stream.cancel());
```

Comment:
@grouma - intentionally unawaited?

think this has anything to do with our force kill on travis?